static/main.css

```diff
@@ -52,4 +53,8 @@ hr {
 .toggle-light {
     color: #ffffff;
     background-color: #072540;
-}
\ No newline at end of file
+}
+
+.term {

```

Comment:
Appreciations for your contribution, and Happy Hacktoberfest! A couple of suggestions/fixes here: 

On this line, the `.term` class is not used in `/layout/terms` in the single or index html files, which causes this edit not to work. We're using the class "term container", which should be changed to `term-container` for code consistency and for a CSS-friendly class that we can then set attributes for. I tried the margin as set using `term-container` as a class across files, and found that this margin breaks our centered titles and does not result in the spacing that we're looking for. 

To submit this change, please change the class name to one that is consistent across files (maybe `term-container`), and implement spacing that is consistent across devices. Make sure to test in Hugo before resubmitting! I'm happy to answer questions if you need.

Reply:
I've seen the `.container` class is used by Bootstrap, so removing it will cause breakages.

I intended for there to be some whitespace between the terms on the index page, to make it easier to see where one ends and the next one begins. That's less of a problem on the single pages, where the content is on its own.

What if I add an additional class for the terms on the index page? So it would look something like `.container .term .new-class` (I was thinking of `.listing`, but could be another name!)

Reply:
Ah- thanks for the follow up! Totally understand- don't want to break Bootstrap's conventions. 

You're right- we'd only need whitespace changes for the index page! I think adding an additional class sounds like a great solution. 

Excited to see your amended commits and our new, more fluid dev glossary! Thanks again for your contribution- it's greatly appreciated!